.github/workflows/e2e.maven.workflow_dispatch.main.default.slsa3.yml

```
@@ -0,0 +1,98 @@
+name: Maven e2e test - simple # TODO: Remove name
+on:
+  - workflow_dispatch
+# TODO: Un-comment this     
+#  schedule:
+#    - cron: "0 6 * * *"
+#  workflow_dispatch:
+#
+permissions: read-all
+
```

nit: add a `concurrency` field.
```suggestion

concurrency: "e2e-maven-workflow_dispatch-main-default-slsa3"

```